app/grandchallenge/core/management/commands/init_gc_demo.py

```diff
@@ -458,18 +470,25 @@ def _create_user_tokens(self):
             "retina": "f1f98a1733c05b12118785ffd995c250fe4d90da",
             "algorithmuser": "dc3526c2008609b429514b6361a33f8516541464",
             "readerstudy": "01614a77b1c0b4ecd402be50a8ff96188d5b011d",
+            "retina_import_user": "e8db90bfbea3c35f40b4537fdca9b3bf1cd78a51",
         }
 
         out = f"{'*' * 80}\n"
         for user, token in user_tokens.items():
             salt = crypto.create_salt_string()
             digest = crypto.hash_token(token, salt)
+            if user == "retina_import_user":
+                user = get_user_model().objects.get(
+                    username=settings.RETINA_IMPORT_USER_NAME
+                )
+            else:
+                user = self.users.get(user)
```

Comment:
I don't understand why this doesn't work for the retina import user?

Reply:
The retina import user is created elsewhere: https://github.com/comic/grand-challenge.org/blob/ba63026bc1024eac7233b6a677ba1da780387670/app/grandchallenge/retina_importers/apps.py#L6-L11

Reply:
Isn't it added to the users dictionary here? https://github.com/comic/grand-challenge.org/pull/1753/files/8ef178ef1b0f135ef3eb85818c84cfc4e6538f53#diff-2aa3920180b8235fb825e66d37bc3a291ec820b8d7111384a42d6d2d81eea9deR100

Reply:
Otherwise I'm not sure why R100 is there

Reply:
Ah you're right! I'll fix that